nimble/syscfg.yml

```diff
@@ -81,7 +81,7 @@ syscfg.defs:
             This allows to configure maximum size of advertising data and
             scan response data used in LE Advertising Extensions.
             Valid range 31-1650.
-        value: 31
+        value: 251
```

Comment:
if we want to change this, it should be separate patch.

Reply:
also if this is required by some feature we should better make 251 conditional depending on that feature, this way default configuration is not changed